packages/superset-ui-connection/test/callApi/callApi.test.ts

```diff
@@ -437,7 +439,7 @@ describe('callApi()', () => {
     });
   });
 
-  it('rejects if the request throws', () => {
+  it('rejects after retrying thrice if the request throws', () => {
```

Comment:
could update the tests to match the retry config variables specified if we expose them

Reply:
this is pretty awesome tho that the throwing is delayed

Reply:
updated